src/main/bash/NodeLimit/limit_docker.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
```

Comment:
Why do we need this file?

Reply:
I forgot to remove this file entirely, I previously made another file since container cannot run sudo, but as we wouldn't run it in the container, it is no longer needed